README.md

```diff
@@ -209,12 +209,13 @@ However some of the Prisma scalars do not have a natural standard representation
 
 **Prisma Standard Scalar to GraphQL Custom Scalar Mapping**
 
-| Prisma     | GraphQL    | Nexus `t` Helper | GraphQL Scalar Implementation                                     |
```

Comment:
Let's add an "Additional Info" column and include MDN links like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt.